biome.hcl

```diff
@@ -25,13 +25,17 @@ version "1.6.3" "1.6.4" "1.6.4-nightly.a995ada" "1.6.4-nightly.abab451" "1.7.0"
         "1.9.0" "1.9.1-nightly.12688b6" "1.9.1" "1.9.2" "1.9.3-nightly.75b4387" "1.9.3"
         "1.9.4" "1.9.5-nightly.92879ae" "1.9.5-nightly.4713c52" "1.9.5-nightly.ff02a0b"
         "1.9.5-nightly.c0cccb2" "1.9.5-nightly.81fdedb" {
+  source = "https://github.com/biomejs/biome/releases/download/cli%2Fv${version}/biome-${os}-${arch_}"
+}
+
+version "2.0.1" "2.0.4" "2.0.5" "2.0.6" "2.1.0" "2.1.1" {
   auto-version {
     github-release = "biomejs/biome"
-    version-pattern = "^cli/v(.*)"
+    version-pattern = "^@biomejs/biome@(.*)"
     ignore-invalid-versions = true
   }
 
-  source = "https://github.com/biomejs/biome/releases/download/cli%2Fv${version}/biome-${os}-${arch_}"
+  source = "https://github.com/biomejs/biome/releases/download/%40biomejs/biome%40${version}/biome-${os}-${arch_}"
```

Comment:
Can you run `hermit add-digests biome.hcl` to add sha256 checksums for the new version?